content/ops/configuration-management.md

```diff
@@ -76,13 +81,3 @@ context to request a review is preferred.
 ### When reviewing a PR, should the change be tested locally?
 
 Whenever possible, the proposed changes should be tested locally. Because of the nature of many of the cloud.gov repositories and deployment environments, local testing is not always possible or practical. Visual code review, however, is always required.
-
-## What if a configuration changed and it is not in Configuration Management?
-If possible Configuration Management tools need to be set up to always roll back to a known state. Other than that, these tools need to be able to "recreate" all settings from the known configurations.
-
-
-### Continuous Monitoring
-
-FedRAMP requires Cloud Service Providers (CSP) to continuously monitor and report vulnerabilities on the system.
-
-cloud.gov adopts the [FedRAMP Continuous Monitoring Strategy Guide](https://www.fedramp.gov/files/2015/03/FedRAMP-Continuous-Monitoring-Strategy-Guide-v2.0-3.docx) as its Continuous Monitoring process.
```

Comment:
I'm not sure it makes sense in general to talk about our Continuous Monitoring Strategy on our Configuration Management Plan page! I suspect what we should do eventually is make a section of our ops documentation for our important team policy docs like these, so that they're close to each other and distinguished from our "we're writing this down to help us get work done" docs.